src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -23,6 +23,7 @@ import { ChromeNavControl, ChromeNavLink } from '../..';
 import { AppCategory, NavGroupType } from '../../../../types';
 import { InternalApplicationStart } from '../../../application/types';
 import { HttpStart } from '../../../http';
+import { OnIsLockedUpdate } from './';
```

Comment:
```suggestion
```
Seems like we are resolving the conflict incorrectly, this line should be removed.